arch/platform/cooja/dev/cooja-radio.c

```diff
@@ -233,24 +223,47 @@ radio_send(const void *payload, unsigned short payload_len)
     return RADIO_TX_ERR;
   }
 
-  /* Transmit on CCA */
-#if COOJA_TRANSMIT_ON_CCA
-  if(send_on_cca && !channel_clear()) {
-    return RADIO_TX_COLLISION;
+  if(radio_was_on) {
+    ENERGEST_SWITCH(ENERGEST_TYPE_LISTEN, ENERGEST_TYPE_TRANSMIT);
+  } else {
+    /* Turn on radio temporarily */
+    simRadioHWOn = 1;
+    ENERGEST_ON(ENERGEST_TYPE_TRANSMIT);
```

Comment:
I think, at this point, the radio is in the listening state. Then, it will perform RX-TX turnaround.